workflow/gwss/rules/gwss-rules.smk

```diff
@@ -46,6 +46,7 @@ rule h12_calibration:
         "logs/h12_calibration/{cohort}.log",
     shell:
         """
+        sleep "$((1+RANDOM%10)).$((RANDOM%999))"
```

Comment:
Add these random sleep commands to try and prevent papermill launch race conditions fighting for the same socket.